.github/workflows/find-old-brew-prs.yml

```diff
@@ -0,0 +1,38 @@
+# Check if we have old Homebrew PRs hanging around unmerged
+name: Check for Old Homebrew PRs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  old-prs:
+    name: Check for old PRs in brew core
+    runs-on: ubuntu-22.04
+    services:
+      datadog-agent:
+        image: datadog/agent:latest
+        ports:
+          - 8125:8125/udp
+        env:
+          DD_API_KEY: ${{ secrets.SEMGREP_DD_API_KEY }}
+          DD_INSIDE_CI: "true"
+          DD_HOSTNAME: "none"
+          DD_DOGSTATSD_NON_LOCAL_TRAFFIC: "1"
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check for Old PRs
+        run: |
+          getPrAge() {
+            # Format (per-line): test.semgrep.pr.age:783514.7839610577|g|#6058
+            gh --repo homebrew/homebrew-core pr list --state open -S semgrep --json number,createdAt \
+              --jq 'now as $date | .[] | [(.createdAt | $date - fromdateiso8601 | tostring), .number] | "semgrep.homebrew.pr.age:" + join("|g|#pr-number:")'
+          }
+
+          PR_AGES=$(getPrAge)
+
+          while IFS= read -r metric_datagram; do
+            echo "$metric_datagram"
+            echo -n "$metric_datagram" | nc -4u -w0 127.0.0.1 8125
+            sleep 10
```

Comment:
I think the agent will be terminated gracefully when the job ends and will have time to flush metrics even without this

Reply:
This was put in place to prevent data from writing too quickly and sends to fail - however I'll merge this PR as is and continue investigating separately.